src/commands.ts

```diff
@@ -210,7 +210,7 @@ export const insertFileAsFigure = (
   }
   const figure = state.schema.nodes.figure.createAndFill({
     label: file.name,
-    src: file.link,
+    src: file.id,
```

Comment:
are you sure this is right?

Reply:
Yes, everything should be using id going forward.